abe/helper_functions/email_helpers.py

```diff
@@ -159,45 +162,45 @@ def error_reply(to, error):
     """ Given the error, sends an email with the errors that
     occured to the original sender. """
     server, sent_from = smtp_connect()
-    subject = 'Event Failed to Add'
+    msg = EmailMessage()
     body = "ABE didn't manage to add the event, sorry. Here's what went wrong: \n"
     for err in error.errors:
         body = body + str(err) + '\n'
     body = body + "Final error message: " + error.message
 
-    email_text = """
-    From: {}
-    To: {}
-    Subject: {}
-
-    {}
-    """.format(sent_from, to, subject, body)
-
-    send_email(server, email_text, sent_from, to)
+    email_text = f"""{body}
```

Comment:
Change to `email_text = body`, unless you need the newline in which case `= body + '\n'`. Or, eliminate `email_text` altogether, and set `msg.set_content(body)` below.